### DISCUSSION TITLE
Add new post from other page

### DISCUSSION THREAD

**Discussion by yussefbens** [aeharding/voyager#853] [Q&A] [answered]
Hi Alex, I want to thank you for this amazing project. I just have one question please, I'm trying to implement a new feature which is adding new posts from a new tab not from the community page's MoreActions, just like the new post tab in reddit, I tried to call **presentPostEditor($community_name)** but I got the error of community not found, I know I have to do the request to get the community details but I don't know what I need to change to do that.

**Comment by aeharding** [comment]
Hi @yussefbens, if you push your code it will be easier for me (or others) to check.

btw I'm converting this to a discussion :)

**Comment by yussefbens** [comment, marked as answer]
for anyone who search for a solution, just add `dispatch(getCommunity(community_name));` to make it work. thank you alex.

**Reply by aeharding** [reply]
Okay glad you figured it out!
